spec/index.bs

```diff
@@ -3842,42 +3840,25 @@ based on the vertex position output. The depth testing and stencil operations ca
+                |features|) succeeds.
+            - If |descriptor|.{{GPURenderPipelineDescriptor/rasterizerState}}.{{GPURasterizerStateDescriptor/sampleCount}} is 1:
+                - |descriptor|.{{GPURenderPipelineDescriptor/outputMergerState}}.{{GPUOutputMergerStateDescriptor/alphaToCoverageEnabled}} is `false`,
             - If |descriptor|.{{GPURenderPipelineDescriptor/fragmentStage}} is not `null`:
                 - [$validating GPUProgrammableStageDescriptor$]({{GPUShaderStage/FRAGMENT}},
                     |descriptor|.{{GPURenderPipelineDescriptor/fragmentStage}},
                     |descriptor|.{{GPUPipelineDescriptorBase/layout}}) succeeds.
-            - If |descriptor|.{{GPURenderPipelineDescriptor/fragmentStage}} is `null`:
-                - |descriptor|.{{GPURenderPipelineDescriptor/colorStates}} is empty.
-            - |descriptor|.{{GPURenderPipelineDescriptor/colorStates}}.length is less than
-                or equal to 4.
-            - For each |colorState| layout descriptor in the list
-                |descriptor|.{{GPURenderPipelineDescriptor/colorStates}}:
-                - [$validating GPUColorStateDescriptor$](|colorState|,
-                    |descriptor|.{{GPURenderPipelineDescriptor/fragmentStage}}) succeeds.
-            - [$validating GPURasterizationStateDescriptor$](|descriptor|.{{GPURenderPipelineDescriptor/rasterizationState}},
-                |features|) succeeds.
             - if |descriptor|.{{GPURenderPipelineDescriptor/depthStencilState}} is not `null`:
                 - [$validating GPUDepthStencilStateDescriptor$](|descriptor|.{{GPURenderPipelineDescriptor/depthStencilState}}) succeeds.
-            - [$validating GPUVertexStateDescriptor$](|descriptor|.{{GPURenderPipelineDescriptor/vertexState}},
-                |descriptor|.{{GPURenderPipelineDescriptor/vertexStage}}) succeeds.
-            - If |descriptor|.{{GPURenderPipelineDescriptor/alphaToCoverageEnabled}} is `true`:
-                - |descriptor|.{{GPURenderPipelineDescriptor/sampleCount}} is greater than 1.
-            - If the output SV_Coverage semantics is [=statically used=] by
-                |descriptor|.{{GPURenderPipelineDescriptor/fragmentStage}}:
-                - |descriptor|.{{GPURenderPipelineDescriptor/alphaToCoverageEnabled}} is `false`.
-            - If |descriptor|.{{GPURenderPipelineDescriptor/primitiveTopology}} is:
-                <dl class="switch">
-                    : {{GPUPrimitiveTopology/"line-strip"}} or
-                        {{GPUPrimitiveTopology/"triangle-strip"}}
-                    :: |descriptor|.{{GPURenderPipelineDescriptor/vertexState}}.{{GPUVertexStateDescriptor/indexFormat}}
-                        is not `undefined`
-                    : Otherwise
-                    :: |descriptor|.{{GPURenderPipelineDescriptor/vertexState}}.{{GPUVertexStateDescriptor/indexFormat}}
-                        is `undefined`
-                </dl>
```

Comment:
Are these rules about indexFormat intentionally removed?

Reply:
they are moved to "validating GPUPrimitiveAssemblerState"